CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Treat legacy messages as text instead of html
 * Fix a bug causing linter messages to briefly disappear and reappear
 * Fix memory leak (Markers weren't getting cleaned up)
+* Use the same filename as the source when generating a temporary file (Fixes #585)
```

Comment:
I don't see any reason not to. The changelog seemed incomplete and kind of arbitrary without listing why this was done